### ISSUE TITLE
How to build project?

### ISSUE THREAD

**Issue by erikprice** [facebook/SocketRocket#369]
I feel like a dummy for even asking this, but how do I build the SocketRocket project? I have a couple of small patches that my own product depends on, and I'd like to submit them upstream, but it's unclear to me how I would actually run the SocketRocket test suite. There are build failures when I try to build in Xcode (one being that `SocketRocket-prefix.pch` is no longer found, but is still specified as a pch file in the settings), and when I try to run `make` the makefile doesn't seem to be able to find all of the source code files.

I assume I must be doing something wrong?

**Comment by nlutsenko** [comment]
Hey @erikprice, I am excited to see those contributions!
The best way to get started with development right now would be to:
- Clone the project and checkout `master` branch
- Open SocketRocket.xcodeproj
- Target the iOS Static Library
- Build & Test aka `CMD + U`

I am working right now on making sure that everyone can run tests and that they run in a CI as well,
but at the current stage there might be some rough edges around installing Autobahn dependencies (the set of web socket tests that we use).

Let me know if you run into any issues going through the process and we'll figure it out together!

**Comment by erikprice** [comment]
Those are the steps I had tried earlier, and building fails due to the `.pch`:
```
clang: error: no such file or directory: '/Users/erikprice/work/SocketRocket/SocketRocket/SocketRocket-Prefix.pch'
clang: error: no input files
```
By removing this from the build settings, I was able to get the project to build successfully, but then the tests failed because Xcode could not find them.

That was earlier today, which prompted me to file this issue. However, just now I've cleaned and nuked the DerivedData, and gone through the above steps again, and it does build and test once I remove the `.pch` from the build settings. "Tests succeeded", although the only test is `(null)`:

![screen shot 2016-05-10 at 16 53 03](https://cloud.githubusercontent.com/assets/3916/15162197/b5ad5830-16cf-11e6-9f6b-e19ad7179879.png)
